packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -159,7 +155,8 @@ export class InteractionManager extends BaseManager<InteractionTypes, Interactio
 
     public constructor(
         private readonly keyboardOptions: { readonly enabled: boolean },
-        private readonly domManager: DOMManager
+        private readonly domManager: DOMManager,
+        private readonly canvasElement: HTMLCanvasElement
```

Comment:
I'd prefer we didn't pass the `canvasElement` in here as it's implicitly hardcoding the expected DOM structure and relations within that here.

Can't we have the `DOMManager` understand where the appropriate places are to attach listeners to the DOM structure its responsible for?